equinix/fabric_connection_read_schema.go

```diff
@@ -0,0 +1,708 @@
+			Description: "Created by Date and Time",
+		},
+		"updated_by": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Updated by User Key",
+		},
+		"updated_by_full_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Updated by User Full Name",
+		},
+		"updated_by_email": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Updated by User Email Address",
+		},
+		"updated_date_time": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Updated by Date and Time",
+		},
+		"deleted_by": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Deleted by User Key",
+		},
+		"deleted_by_full_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Deleted by User Full Name",
+		},
+		"deleted_by_email": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Deleted by User Email Address",
+		},
+		"deleted_date_time": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Deleted by Date and Time",
+		},
+	}
+}
+
+func readOrderSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"purchase_order_number": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Purchase order number",
+		},
+		"billing_tier": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Billing tier for connection bandwidth",
+		},
+		"order_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Order Identification",
+		},
+		"order_number": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Order Reference Number",
+		},
+	}
+}
+
+func readAccountSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"account_number": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Account Number",
+		},
+		"account_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Account Name",
+		},
+		"org_id": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Customer organization identifier",
+		},
+		"organization_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Customer organization name",
+		},
+		"global_org_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Global organization identifier",
+		},
+		"global_organization_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Global organization name",
+		},
+		"global_cust_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Global Customer organization identifier",
+		},
+	}
+}
+
+func readNotificationSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"type": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Notification Type",
```

Comment:
Do we have any hint on the types to expect? Let's include them in the description if so.

Reply:
It is read resource, but taken care.